Task_DEV-8/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace task_DEV_8
+{
+    /// <summary>
+    /// Entry point of programm. Input two matrix and multiply them,if it's possible
+    /// </summary>
+    class Program
+    {
+        const int CoefficientFirstSecond = 1;
+        const int CoefficientSecondFirst = 2;
+        static void Main(string[] args)
+        {
+            Inputer inputer = new Inputer();
+            Checker checker = new Checker();
+            Multiplier multiplier = new Multiplier();
+            Printer printer = new Printer();
+            double[,] firstMatrix = inputer.Input();
```

Comment:
It will be better to create separate class for Matrix, and Multiply method in it